.circleci/Dockerfile

```diff
@@ -24,27 +40,11 @@ RUN     cd /tmp && \
 
 RUN     mkdir -p /tmp/apiserver && \
         cd /tmp/apiserver && \
-        wget https://storage.googleapis.com/kubernetes-release/release/v1.7.4/bin/linux/amd64/kube-apiserver && \
+        curl -L -o kube-apiser https://storage.googleapis.com/kubernetes-release/release/${K8S_VER}/bin/linux/amd64/kube-apiserver && \
```

Comment:
should this not output kube-apiserver?

Reply:
Sorry, I replaced wget with curl (after I built and tested the image..), for consistency.